flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+        "x86_64"
```

Comment:
disappointing. can't you build from source?

Reply:
Hi, its flutter app